entrypoint.sh

```diff
@@ -9,6 +9,9 @@ cvmfsexec_root=/cvmfsexec
 cvmfsexec_tarball=/cvmfsexec.tar.gz
 cvmfsexec_local_config=$cvmfsexec_root/dist/etc/cvmfs/default.local
 
+# Delete any whitespace in CVMFSEXEC_REPOS
+CVMFSEXEC_REPOS=$(tr -d $' \t\n' <<<"$CMFSEXEC_REPOS")
```

Comment:
What's the `$` do here?

Reply:
`$'...'` interprets backslashed characters:
```
$ echo x\tx
xtx
$ echo x\\tx
x\tx
$ echo 'x\tx'
x\tx
$ echo "x\tx"
x\tx
$ echo $'x\tx'
x	x
```
not to be confused with `$"..."` which translates the message to the current language.